projects/1.16/assets/born-in-chaos/born_in_chaos_v1/lang/zh_cn.json

```
@@ -0,0 +1,196 @@
+  "block.born_in_chaos_v1.scorched_planks": "焦炭木",
+  "block.born_in_chaos_v1.scorched_wood": "焦炭木",
+  "block.born_in_chaos_v1.smoldering_scorched_log": "阴燃焦炭原木",
+  "block.born_in_chaos_v1.smoldering_scorched_wood": "焦炭木",
+  "block.born_in_chaos_v1.stained_black_argillite_brick_n": "§4诅咒染色粘板岩砖",
+  "block.born_in_chaos_v1.stained_black_argillite_brick": "染色粘板岩砖",
+  "effect.block_break": "破块",
+  "effect.cursed_mark": "诅咒标记",
+  "effect.furious_rampage": "猖獗狂暴",
+  "effect.light_rampage": "光之狂暴",
+  "effect.magic_depletion": "",
+  "effect.medium_rampage": "中等狂暴",
+  "effect.rampant_rampage": "终极狂暴",
+  "effect.strong_rampage": "强化狂暴",
+  "effect.stun": "击晕",
+  "entity.born_in_chaos_v1.baby_sk_pow": "骷髅小鬼",
+  "entity.born_in_chaos_v1.baby_skeleton_n": "小骷髅",
+  "entity.born_in_chaos_v1.baby_skeleton": "小骷髅",
+  "entity.born_in_chaos_v1.baby_skkl": "小骷髅",
+  "entity.born_in_chaos_v1.bone_imp_n": "骷髅小鬼",
+  "entity.born_in_chaos_v1.bone_imp": "骷髅小鬼",
+  "entity.born_in_chaos_v1.bonescaller_n": "骷髅召唤师",
+  "entity.born_in_chaos_v1.bonescaller": "骷髅召唤师",
+  "entity.born_in_chaos_v1.controlled_baby_skeleton": "被控制的小骷髅",
+  "entity.born_in_chaos_v1.dark_vortex_n": "暗影漩涡",
+  "entity.born_in_chaos_v1.dark_vortex": "暗影漩涡",
+  "entity.born_in_chaos_v1.decaying_zombie_n": "腐尸",
+  "entity.born_in_chaos_v1.decaying_zombie": "腐尸",
+  "entity.born_in_chaos_v1.decrepit_skeleton_n": "骨架",
+  "entity.born_in_chaos_v1.decrepit_skeleton": "骨架",
+  "entity.born_in_chaos_v1.diamond_thermite": "钻石甲虫",
+  "entity.born_in_chaos_v1.dread_hound_n": "骇狼",
+  "entity.born_in_chaos_v1.dread_hound": "骇狼",
+  "entity.born_in_chaos_v1.fallen_chaos_knight": "堕落混沌骑士",
+  "entity.born_in_chaos_v1.firelight_n": "焰芒",
+  "entity.born_in_chaos_v1.firelight": "焰芒",
+  "entity.born_in_chaos_v1.nightmare_stalker_n": "潜伏者",
+  "entity.born_in_chaos_v1.nightmare_stalker": "潜伏者",
+  "entity.born_in_chaos_v1.phantom_creeper_form": "幻影苦力怕",
+  "entity.born_in_chaos_v1.phantom_creeper_n": "幻影苦力怕",
+  "entity.born_in_chaos_v1.phantom_creeper": "幻影苦力怕",
+  "entity.born_in_chaos_v1.pumpkin_spirit": "南瓜灵魂",
+  "entity.born_in_chaos_v1.restless_spirit_n": "急躁幽灵",
+  "entity.born_in_chaos_v1.restless_spirit": "急躁幽灵",
+  "entity.born_in_chaos_v1.s_bonescaller_n": "高阶骷髅召唤师",
+  "entity.born_in_chaos_v1.scarlet_persecutor": "绯红幽灵",
+  "entity.born_in_chaos_v1.seared_spirit": "火焚",
+  "entity.born_in_chaos_v1.skeleton_bomb_n": "自爆骷髅",
+  "entity.born_in_chaos_v1.skeleton_bomb": "自爆骷髅",
+  "entity.born_in_chaos_v1.skeleton_thrasher_n": "重击者",
+  "entity.born_in_chaos_v1.skeleton_thrasher": "重击者",
+  "entity.born_in_chaos_v1.supreme_bonecaller": "高阶骷髅召唤师",
+  "entity.born_in_chaos_v1.supreme_bonescaller_s_2": "高阶骷髅召唤师",
+  "entity.born_in_chaos_v1.wither_strider_n": "凋零水黾",
+  "entity.born_in_chaos_v1.wither_strider": "凋零水黾",
+  "entity.born_in_chaos_v1.zombiesina_barrel": "木桶僵尸",
+  "entity.born_in_chaos_v1.zombieswith_door_n": "铁桶门骑士",
+  "entity.born_in_chaos_v1.zombieswith_door": "铁桶门骑士",

```

| **审查级别** |
|--------------|
| ℹ️ 信息 / 不阻止合并  |
原文：**Door Knight**
```suggestion
  "entity.born_in_chaos_v1.zombieswith_door": "铁桶门骑士",
```
虽然名字里没“铁桶”，但它确实头上盖了个铁桶避免自己燃烧